.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+to:
```

Comment:
Please remove this one as well. We can follow up with any contribution changes like this